Softeq.XToolkit.Chat.iOS/ViewControllers/ChatDetailsViewController.cs

```diff
@@ -57,6 +60,30 @@ protected override void DoAttachBindings()
             Bindings.Add(this.SetBinding(() => ViewModel.ChatName, () => _chatDetailsHeaderView.ChatNameField.Text));
             Bindings.Add(this.SetBinding(() => ViewModel.MembersCountText, () => _chatDetailsHeaderView.ChatMembersCount));
         }
-    }
-}
 
+        private class ParticipantsTableViewDelegate : UITableViewDelegate
+        {
+            private readonly ChatDetailsViewModel _viewModel;
+
+            public ParticipantsTableViewDelegate(ChatDetailsViewModel viewModel)
+            {
```

Comment:
depends on VM lifetime (register in IoC), use weakRef of VM

Reply:
discussed